v3.13.3 is not in Docker Hub, nor GitHub Container Registry

**Describe the bug**

Version 3.13.3 was not deployed to any artifact repository because the [build failed](https://github.com/github/super-linter/runs/1294033734). I've seen the build failing many times because of this exact issue. Time to fix it? :)

**To Reproduce**
Steps to reproduce the behavior:

1. `docker run ghcr.io/github/super-linter:v3.13.2`

**Expected behavior**

Run `ghcr.io/github/super-linter:v3.13.2`.

**Logs**

```shell
"super-linter": Error response from daemon: name unknown
```

**Additional context**

Complete build log: https://github.com/github/super-linter/runs/1294033734

Relevant extract:

```shell
Step 41/76 : RUN wget --tries=5 -O phive.phar https://phar.io/releases/phive.phar     && wget --tries=5 -O phive.phar.asc https://phar.io/releases/phive.phar.asc     && gpg --keyserver pool.sks-keyservers.net --recv-keys 0x9D8A98B29B2D5D79     && gpg --verify phive.phar.asc phive.phar     && chmod +x phive.phar     && mv phive.phar /usr/local/bin/phive     && rm phive.phar.asc     && phive install --trust-gpg-keys 31C7E470E2138192,CF1A108D0E7AE720,8A03EA3B385DBAA1
 ---> Running in 29bc54cc3eed
Connecting to phar.io (188.94.27.6:443)
Connecting to github.com (140.82.114.4:443)
Connecting to github-production-release-asset-2e65be.s3.amazonaws.com (52.216.102.91:443)
saving to 'phive.phar'
phive.phar           100% |********************************|  187k  0:00:00 ETA
'phive.phar' saved
Connecting to phar.io (188.94.27.6:443)
Connecting to github.com (140.82.112.3:443)
Connecting to github-production-release-asset-2e65be.s3.amazonaws.com (54.231.120.219:443)
saving to 'phive.phar.asc'
phive.phar.asc       100% |********************************|   854  0:00:00 ETA
'phive.phar.asc' saved
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: keyserver receive failed: Address not available
The command '/bin/sh -c wget --tries=5 -O phive.phar https://phar.io/releases/phive.phar     && wget --tries=5 -O phive.phar.asc https://phar.io/releases/phive.phar.asc     && gpg --keyserver pool.sks-keyservers.net --recv-keys 0x9D8A98B29B2D5D79     && gpg --verify phive.phar.asc phive.phar     && chmod +x phive.phar     && mv phive.phar /usr/local/bin/phive     && rm phive.phar.asc     && phive install --trust-gpg-keys 31C7E470E2138192,CF1A108D0E7AE720,8A03EA3B385DBAA1' returned a non-zero code: 2
2020-10-22 17:38:46 [FATAL ]   failed to [build] Dockerfile!
Error: Process completed with exit code 1.
```

Possible workaround, as discussed in https://github.com/nodejs/docker-node/issues/380#issuecomment-348464695

Instead of just running:

```shell
gpg --keyserver pool.sks-keyservers.net --recv-keys 0x9D8A98B29B2D5D79
```

use something like:

```shell
( gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$GPG_KEYS" \
  || gpg --keyserver pgp.mit.edu --recv-keys "$GPG_KEYS" \
  || gpg --keyserver keyserver.pgp.com --recv-keys "$GPG_KEYS" )
```

Thoughts?

--- a/Dockerfile
+++ b/Dockerfile
@@ -157,7 +157,10 @@ RUN curl --retry 5 --retry-delay 5 -sL https://cpanmin.us/ | perl - -nq --no-wge
 ##############################
 RUN wget --tries=5 -O phive.phar https://phar.io/releases/phive.phar \
     && wget --tries=5 -O phive.phar.asc https://phar.io/releases/phive.phar.asc \
-    && gpg --keyserver pool.sks-keyservers.net --recv-keys 0x9D8A98B29B2D5D79 \
+    && PHAR_KEY_ID="0x9D8A98B29B2D5D79" \
+    && ( gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$PHAR_KEY_ID" \
+    || gpg --keyserver pgp.mit.edu --recv-keys "$PHAR_KEY_ID" \
+    || gpg --keyserver keyserver.pgp.com --recv-keys "$PHAR_KEY_ID" ) \
     && gpg --verify phive.phar.asc phive.phar \
     && chmod +x phive.phar \
     && mv phive.phar /usr/local/bin/phive \

